ehrapy/tools/feature_ranking/_feature_importances.py

```diff
@@ -120,22 +120,26 @@ def rank_features_supervised(
     input_data = data[input_features]
     labels = data[predicted_feature]
 
+    x_train, x_test, y_train, y_test = train_test_split(input_data, labels, test_size=test_split_size)
```

Comment:
sorry for hopping in - what do you think about adding a random_state here? :)

Reply:
Good idea, thanks!